superdesk/publish/formatters/nitf_formatter.py

```diff
@@ -190,18 +190,19 @@ def _parse_children(self, html_elem, root_elem):
 
         return html_elem
 
-    def html2nitf(self, html_elem, root_elem=None):
+    def html2nitf(self, html_elem, root_elem=None, attr_reset=None):
         """Convert HTML elements to NITF compatible elements
 
         :param ET.Element: HTML to clean/transform
         :param ET.Element: None if its the root element (must be a <div>)
+        :param list attr_reset: attributes to remove if present
```

Comment:
style can be put afterwards by the NITF mapping (e.g. if there where "style='fontstyle:italic'" when <em> is found), that's why I put reset (it's set to nothing before being updated), so I think "blacklist" is not OK, "remove" may do it.